packages/fuel-indexer-lib/src/config/mod.rs

```diff
@@ -380,6 +387,7 @@ impl Default for IndexerArgs {
             rate_limit: defaults::RATE_LIMIT_ENABLED,
             rate_limit_request_count: Some(defaults::RATE_LIMIT_REQUEST_COUNT),
             rate_limit_window_size: Some(defaults::RATE_LIMIT_WINDOW_SIZE),
+            replace_indexer: false,
```

Comment:
Let's follow our pattern of using defaults here with `defaults::REPLACE_INDEXER` (or some such)